src/Data/Process.elm

```diff
@@ -59,6 +60,14 @@ type SourceId
     = SourceId String
 
 
+{-| List processes which ids are not part of the provided list of ids
+-}
+available : List Id -> List Process -> List Process
+available alreadyUsedIds =
+    List.filter (\{ id } -> not <| List.member id alreadyUsedIds)
+        >> List.sortBy .name
```

Comment:
Shouldn’t we sort by `displayName` if available and fallback to `name` if there is no display name?

Reply:
Addressed in https://github.com/MTES-MCT/ecobalyse/pull/967/commits/70c97030d692a1f5b6d0349e98d9a89a5273b7bb